version.py

```diff
@@ -21,4 +21,4 @@
 # Typically, adding a dependency only requires a minor version bump, and
 # removing a dependency requires a major version bump.
 
-PROVISION_VERSION = '37.7'
+PROVISION_VERSION = '37.8'
```

Comment:
This needs to be a major bump since `autosize` doesn’t satisfy `jquery-autosize` going backwards.

Reply:
Understood. Changed it to `38.0`.